pennylane/_grad.py

```diff
@@ -525,9 +531,9 @@ def finite_diff(f, N=1, argnum=0, idx=None, delta=0.01):
     -0.372062798810191
     """
     warnings.warn(
-        "The black-box finite_diff function will be deprecated, users can "
-        "instead use qml.gradients.finite_diff to compute the gradient of "
-        "tapes or QNodes. Otherwise, manual implementation is required.",
+        "The black-box finite_diff function is deprecated and will be removed in an upcoming release. "
+        "To compute quantum gradients of tapes or QNodes using finite-differences "
+        "please see qml.gradients.finite_diff.",
```

Comment:
Slightly modified to avoid using the term 'users', and also to change the tense (will be deprecated -> is deprecated + will be removed)

Reply:
(cc @Jaybsoni)

Reply:
Ahh this makes more sense! I thought being deprecated meant that it was already removed ! Thanks for double checking this !